android/app/src/main/java/fr/coopcycle/MainApplication.java

```diff
@@ -26,6 +33,13 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
+            new StripeReactPackage(),
```

Comment:
Is it a `react-native` good practice to do this?

Reply:
I don't know, these files were automatically updated when I ran `link`/`start` 🤷‍♂️

Reply:
Which version of `react-native` are you using?

Reply:
```
$ react-native --version
react-native-cli: 2.0.1
react-native: 0.53.2
```

Reply:
Yep, there is a problem when running `react-native-link`, it duplicates some lines in the android project

Reply:
OK, I can manually remove duplicated lines in the android source, unless you know of a better solution?

Reply:
Yes please remove those lines. 
I need some time to understand the cause. I suspect the Gradle plugin because it affects only Android.